package.json

```diff
@@ -130,6 +136,7 @@
     "check-node-version": "^4.2.1",
     "chttps": "^1.0.6",
     "css-loader": "^7.1.2",
+    "d3-geo": "^3.1.1",
```

Comment:
So `d3-geo` is upgraded from v1 to v3. 
This looks fine. But I'm wondering if it was necessary? 
Also if we could do that in a separate PR?

Reply:
Also `d3-geo` is used in the source code. So it should be listed in dependencies not dev-dependencies.